test_scripts/API/SendLocation/commonSendLocation.lua

```diff
@@ -25,33 +25,36 @@ commonSendLocation.minTimeout = 500
 commonSendLocation.DEFAULT = "Default"
 commonSendLocation.successResultCodes = {
   "SUCCESS",
-  "RETRY",
-  "WARNINGS",
   "SAVED"
 }
 
 commonSendLocation.failureResultCodes = {
-  "UNSUPPORTED_REQUEST",
   "UNSUPPORTED_RESOURCE",
   "DISALLOWED",
   "REJECTED",
   "ABORTED",
-  "IGNORED",
+  "INVALID_DATA",
+  "INVALID_ID",
```

Comment:
@Itileda `INVALID_ID` should be in `unexpectedResultCodes` table according to MOBILE_API

Reply:
@AKalinich-Luxoft , fixed in a1dd9de8f0fb3a7642d9a9d9897aff95e763f59e